LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/LnsMessageType.cs

```diff
@@ -22,4 +25,40 @@ internal enum LnsMessageType
 
         */
     }
+
+    internal static class LnsMessageTypeExtensions
+    {
+        internal static bool TryParseLnsMessageType(string s, out LnsMessageType? lnsMessageType)
+        {
+            lnsMessageType = s switch
+            {
+                "version" => LnsMessageType.Version,
+                "router_config" => LnsMessageType.RouterConfig,
+                "jreq" => LnsMessageType.JoinRequest,
+                "updf" => LnsMessageType.UplinkDataFrame,
+                "dntxed" => LnsMessageType.TransmitConfirmation,
+                "dnmsg" => LnsMessageType.DownlinkMessage,
+                _ => null
+            };
+            return lnsMessageType is not null;
+        }
+
+        internal static LnsMessageType ParseAndValidate(string s, LnsMessageType? expectedType) =>
```

Comment:
Since this method is only used in one place, why not just have it in-lined there?

Reply:
I would keep it separate for future re-usability and more isolated testing if you don't have a strong opinion on this 👍